ApollosProject/apollos-prototype
# Improves android profile image quality

## DESCRIPTION

### What does this PR do, or why is it needed?

The quality was previously set super low, I believe to try and improve the transit time for the image making a roundtrip to Rock a back. Since that change, we have dramatically improved the UX of waiting for an image to show up, so I think we're save to up the quality a bit to avoid the potato android photos.

![2020-01-27 13 35 04](https://user-images.githubusercontent.com/1637694/73205304-c2193180-410e-11ea-94fc-55cbc342f86d.gif)
![2020-01-27 13 30 20](https://user-images.githubusercontent.com/1637694/73205313-c5142200-410e-11ea-82ed-00c6ed163ab9.gif)


### What design trade-offs/decisions were made?

Went with 70% compression for a decent size/quality tradeoff. 

### How do I test this PR?

1. Open the app (preferably on a physical Android device)
2. Navigate to your profil
3. Tap on the camera icon
4. Tap on "take a photo" 
5. Take a photo.
6. When your profile image returns, ensure it is not washed of color and visibly pixelated. 

### What automated tests did you write?

n/a

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
